jetstream/sponsored-suggestion-placement-phase-2.toml

```diff
@@ -0,0 +1,658 @@
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "Ad clicks per exposed search session"
+description = "Per-client rate of SERP ad clicks per exposed urlbar search session"
+
+[metrics.exposed_ad_clicks_per_session.statistics.bootstrap_mean]
+[metrics.exposed_ad_clicks_per_session.statistics.deciles]
+
+[metrics.exposed_adm_impressions_per_session]
+select_expression = """SAFE_DIVIDE(
+    COUNTIF(exposed_session AND urlbar_fields.has_sponsored_impression),
+    COUNTIF(exposed_session AND new_session_start)
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "Sponsored suggestion impressions per exposed search session"
+description = "Per-client rate of sponsored suggestion impressions per exposed urlbar search session"
+
+[metrics.exposed_adm_impressions_per_session.statistics.bootstrap_mean]
+[metrics.exposed_adm_impressions_per_session.statistics.deciles]
+
+[metrics.exposed_adm_clicks_per_session]
+select_expression = """SAFE_DIVIDE(
+    COUNTIF(
+        exposed_session
+        AND urlbar_fields.event_action = 'engaged'
+        AND urlbar_fields.product_selected_result = 'admarketplace_sponsored'
+    ),
+    COUNTIF(exposed_session AND new_session_start)
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "Sponsored suggestion clicks per exposed search session"
+description = "Per-client rate of sponsored suggestion clicks per exposed urlbar search session"
+
+[metrics.exposed_adm_clicks_per_session.statistics.bootstrap_mean]
+[metrics.exposed_adm_clicks_per_session.statistics.deciles]
+
+[data_sources]
+
+[data_sources.tagged_google_serp_events]
+from_expression = """(
+    SELECT
+        * EXCEPT (is_ad_component),
+        component IN ('ad_carousel', 'ad_image_row', 'ad_link', 'ad_sidebar',
+        'ad_sitelink') AS ad_component
+    FROM `mozdata.firefox_desktop.serp_events`
+    WHERE search_engine = 'google' AND is_tagged
+)"""
+client_id_column = "legacy_telemetry_client_id"
+experiments_column_type = "native"
+submission_date_column = "submission_date"
+friendly_name = "Glean Google tagged SERP events"
+description = "Glean SERP events aggregated by impression, component for tagged Google search"
+
+[data_sources.urlbar_search_sessions]
+friendly_name = "Urlbar search sessions"
+description = "Search sessions initiated in the urlbar including urlbar and tagged Google serp Glean events"
+client_id_column = "legacy_telemetry_client_id"
+submission_date_column = "submission_date"
+experiments_column_type = "native"
+from_expression = """(
+    WITH urlbar AS (
+      -- terminal urlbar events (urlbar sessions)
+      SELECT
+        *
+      FROM
+        `mozdata.firefox_desktop.urlbar_events`
+      WHERE
+        is_terminal
+        AND submission_date BETWEEN '2024-01-16' AND '2024-04-22'
+        AND mozfun.map.get_key(experiments, 'sponsored-suggestion-placement-phase-2') IS NOT NULL
+    ),
+    urlbar_with_exposed AS (
+      -- interleave exposure events with main urlbar events
```

Comment:
Should remove these comments for clarity as we're not working with the exposure events here